server/bleep/src/lib.rs

```diff
@@ -4,10 +4,10 @@
     future_incompatible,
     nonstandard_style,
     rust_2018_idioms,
-    unused_crate_dependencies,
     unused_lifetimes,
     unused_qualifications
 )]
+#![warn(unused_crate_dependencies)]
```

Comment:
What is the crate that makes this necessary? Can we instead make it explicit down below for the config targets it affects?

Reply:
This was originally `#[deny]`, but this is annoying during development. I was testing versions between parser crates, and at some point had a compilation error that I temporarily fixed by using `use crate_name as _;`. This is suboptimal for developer UX. Making this a warning means we still notice when there is an issue, but avoids the situation where WIP code doesn't compile due to ongoing changes.